### PR TITLE
[On pause] Migrate Improve > Design > Positions > Transplant module

### PR DESCRIPTION
**Pull request by zuk3975**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Migrate Improve > Design > Positions > Transplant module.
| Type?         | refacto
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/10568
| How to test?  | 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/14038)

### CODE REVIEW

**Comment by zuk3975** [comment]
@matks, in order to proceed with this migration i need a clarification. Regarding this #14037 issue, i think a lot must be changed, first including the main aim of module transplantation, re-considering allowed changes for user ( as it could be harmfull for the shop), also adding some kind of reset/undo button to revert the changes, this means some design changes might be introduced and so on... Overall It's a mess :smile:

**Comment by matks** [comment]
Let's discuss it here https://github.com/PrestaShop/PrestaShop/issues/14037#issuecomment-497691303

In the meantime, this PR can be paused

**Comment by matks** [comment]
I'm pinging product team on the issue in order to make this PR progress

**Comment by matks** [comment]
According to https://github.com/PrestaShop/PrestaShop/issues/14037#issuecomment-499110042
> If i'm correct, this page had not been anticipated (another one...), so you should add it to our document and change the roadmap according to it.

So this PR will wait, it's not yet to be done according to roadmap

File: src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Position/Blocks/form.html.twig [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+  * International Registered Trademark & Property of PrestaShop SA
+  *#}
+
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{{ form_start(moduleHookForm) }}
+<div class="card">
+  <div class="card-header">
+    {{ 'Positions'|trans({}, 'Admin.Design.Feature') }}
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Improve/Design/Position/Blocks/form.html.twig:31]:
```suggestion
    {{ 'Positions'|trans({}, 'Admin.Navigation.Menu') }}
```

**Comment by PierreRambaud** [comment]
Time to close it and reopen if needed?

**Pull request closed without merging**